dotnet/GspSub.cs

```diff
@@ -0,0 +1,7 @@
+channel.Subscribe("position", msg => {
+    var pos = (GpsPosition) msg.Data;
```

Comment:
Not sure if we can just cast it to get actual `GpsPosition` type, all returned data will be in the form newtonsoft `JToken` format (typical json map), we might have to manually construction `GpsPosition` object from it. Not 100 % sure, but you should check @marto83 ur suggestion would be useful too

Reply:
Something like this might be required, https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_Linq_JToken_ToObject.htm

Reply:
As much as possible I've tried to stay within the budget of 5 lines and 64 columns. The code above assumes we have the`GpPosition` type defined elsewhere.

Reply:
Resolving this conversation so that we can land this work as it's needed for other teams to work on tomorrow.